src/relay/qnn/util.h

```diff
@@ -86,6 +86,12 @@ static inline Expr Requantize(const Expr& data, const Array<IndexExpr>& input_sh
   return RequantizeLower(data, attrs.operator->(), input_shape, out_dtype);
 }
 
+static inline int64_t get_const_int(const tvm::Expr& x) {
+  auto* value_ptr = as_const_int(x);
+  CHECK(value_ptr) << "Expr is not a constant int";
```

Comment:
Could it be CHECK_NOTNULL?

Reply:
Thanks for pointing to this. This one is better but it is a macro and does not support ostream. I think sticking to CHECK(..) should be ok here for the error reporting.